pyccel/ast/literals.py

```diff
@@ -44,8 +44,14 @@ def precision(self, precision):
     def python_value(self):
         """ Get python literal represented by this instance """
 
+    def __repr__(self):
+        return repr(self.python_value)
+
+    def _sympystr(self, printer):
+        return printer.doprint(self.python_value)
+
 #------------------------------------------------------------------------------
-class LiteralTrue(sp_BooleanTrue, Literal):
+class LiteralTrue(Literal, sp_BooleanTrue):
```

Comment:
why do we still have `sp_BooleanTrue`

Reply:
Because nobody said we could remove it? I think you are right. It should be removed. The only thing it might be doing is ensuring that `LiteralTrue` is a singleton, but we don't necessarily want that if we want booleans with different precision. I will remove it